hrjobcontract/js/test/services/utils_test.js

```diff
@@ -0,0 +1,51 @@
+define([
+    'common/angular',
+    'common/angularMocks',
+    'job-contract/app'
+], function (angular, Mock) {
+    'use strict';
```

Comment:
Just realized you are using a 4 space indentation here, please use 2 (aren't we using and editorconfig file for this?)